db-engine/src/main/java/com/splicemachine/db/iapi/types/DataType.java

```diff
@@ -1370,23 +1370,33 @@ public int compare(DataValueDescriptor o1, DataValueDescriptor o2) {
         }
     }
 
-    /**
-     *
-     * 256 gives < 1% normalized rank error
-     *
-     * @return
-     * @throws StandardException
+    /* 256 gives < 1% normalized rank error. Note that this error is independent to data
+     * stream size. Reference:
+     * https://datasketches.apache.org/docs/Quantiles/QuantilesAccuracy.html .
      */
     @Override
     public com.yahoo.sketches.quantiles.ItemsSketch getQuantilesSketch() throws StandardException {
         return com.yahoo.sketches.quantiles.ItemsSketch.getInstance(256,this);
     }
 
+    /* Bigger maxMapSize leads to better accuracy. Reference:
+     * https://datasketches.apache.org/docs/Frequency/FrequentItemsErrorTable.html .
+     * Memory consumption is (18 bytes * currentMapSize). currentMapSize starts from 8 and
+     * gets doubled when the number of items in map is bigger than 0.75 * currentMapSize.
+     * Also, currentMapSize is capped at maxMapSize.
+     */
     @Override
     public com.yahoo.sketches.frequencies.ItemsSketch getFrequenciesSketch() throws StandardException {
-        return new com.yahoo.sketches.frequencies.ItemsSketch(256);
+
+        return new com.yahoo.sketches.frequencies.ItemsSketch(8192);
     }
 
+    /* Bigger nomEntries leads to better accuracy. Reference:
+     * https://datasketches.apache.org/docs/Theta/ThetaErrorTable.html .
+     * Internal hash table size could also start small and grow as needed but it depends
+     * on the value of resizeFactor, or lgRF. Check Util.startingSubMultiple() in Sketch
+     * library for the logic determining the starting size.
+     */
     @Override
     public UpdateSketch getThetaSketch() throws StandardException {
         return UpdateSketch.builder().build(256);
```

Comment:
While in the neighborhood, can we change this one from a magic number as well?

Reply:
Sure, the scope of DB-11481 will cover all the three sketch sizes.